src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/AddCacheWarmerPassTest.php

```diff
@@ -33,7 +33,6 @@ public function testThatCacheWarmersAreProcessedInPriorityOrder()
             ->will($this->returnValue($services));
         $container->expects($this->atLeastOnce())
             ->method('getDefinition')
-            ->with('cache_warmer')
```

Comment:
Not sure what's your goal here Nicolas, but I'd also replace `->expects($this->atLeastOnce())` with `->expects($this->any())` (use a stub instead of a mock). Removing the `with()` call seems wrong, as the test really should expect this particular service definition to be fetched from the container (as opposed to any definiton).